src/txkube/testing/integration.py

```diff
@@ -481,6 +487,74 @@ def check_configmaps(collection):
 
 
 
+class _PodTestsMixin(object):
+    @async
+    @needs(namespace=creatable_namespaces())
+    def test_pod(self, namespace):
+        """
+        ``Pod`` objects can be created and retrieved using the ``create`` and
+        ``list`` methods of ``IKubernetesClient``.
+        """
+        return self._create_list_test(
```

Comment:
`_create_list_test` isn't part of this PR, but that calls [`example()` on the strategy]( https://github.com/LeastAuthority/txkube/blob/cbaedf9dd3307b617edf6d1000161235fc9716b3/src/txkube/testing/integration.py#L869), which is `pods()` here.  You might consider changing that to use [`data.draw`](http://hypothesis.readthedocs.io/en/latest/data.html#drawing-interactively-in-tests), which in turn require that this be decorated with `given`.  Is the concern that doing so will run the test method more than once?